Vagrantfile

```diff
@@ -73,6 +73,10 @@ Vagrant.configure(2) do |config|
     config.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_opensuse-13.2-x86_64_chef-provisionerless.box"
     suse_common config
   end
+  config.vm.define "sles-12" do |config|
+    config.vm.box = "idar/sles12"
```

Comment:
I remember this box being super weird - I see you worked around the media missing issues with those DVD declarations and stuff.... Weird.

What is the licensing situation like for this?